lib/raygun.sync.worker.ts

```diff
@@ -2,16 +2,17 @@ import fs from "fs";
 
 import * as transport from "./raygun.transport";
 import { SendOptions, SendOptionsWithoutCB } from "./types";
+import {IncomingMessage} from "http";
```

Comment:
Move the import up as well - 
```suggestion
import { IncomingMessage } from "http";
```